app/server/docker-compose.yml

```
@@ -15,7 +15,7 @@ services:
     networks:
       - appsmith
     volumes:
-      - ./container-volumes/git-storage:/data/git-storage
+      - /container-volumes/git-storage:/data/git-storage
```

We need the `.` here since we want the path on the host system to be relative to the `docker-compose.yml` file, instead of at the root `/` path. We want a root based path only _inside_ the container, which is `/data/git-storage`.

```suggestion
      - ./container-volumes/git-storage:/data/git-storage
```